### ISSUE TITLE
Excessive line spacing in description text when windowed (Vulkan)

### ISSUE THREAD

**Issue by bryc** [libretro/RetroArch#5452]
## Description
There is excessive line spacing in the item description text when using Vulkan in window mode.

### Expected behavior
The text appears fine when in full-screen mode (using F to toggle):
![image](https://user-images.githubusercontent.com/1408749/30543543-8e3fe9e4-9c51-11e7-847c-27e21d8ce11e.png)

### Actual behavior
When in window mode, there is excessive spacing and it overlaps other text, making the description difficult to read:
![image](https://user-images.githubusercontent.com/1408749/30543529-831df7e0-9c51-11e7-8539-a14a51560574.png)

This does not happen with OpenGL when in window mode.

### Steps to reproduce the bug
1. Set video driver to Vulkan (and reload RetroArch if necessary)
2. Switch from Full-screen mode to Window mode, and observe the description texts, compare the line spacing.

### Bisect Results
This issue existed in 1.6.0 and 1.6.4, I'm not sure if it was ever working properly ever since Vulkan was added.

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.6.7

### Environment information
- OS: Windows 7 Professional x64
- Compiler: N/A

**Comment by fr500** [comment]
It's a problem but it's not related to screen mode but aspect ratio.
Set Window Width / Height to 1280 / 720 and it should look fine.

**Comment by bryc** [comment]
right, so Vulkan can't handle 4:3 ratios.. i guess i can deal in 720p in window for now

**Comment by zoltanvb** [comment]
This does not seem to be an issue nowadays, somewhere along the line it was fixed. Tested with Vulkan and window resized to random ratios - it responded quickly and arrived to a good result. I think it can be closed.
